doc/user_guide/forwardSamplingExample.tex

```diff
@@ -314,11 +313,10 @@ \subsection{Stratified sampling through RAVEN}
 \begin{enumerate}
    \item \textbf{\textit{RunInfo}}:
      \xmlExample{framework/user_guide/ForwardSamplingStrategies/forwardSamplingStratified.xml}{RunInfo}
-   As reported in Section~\ref{sub:EntitiesAndFlow}, the \textit{RunInfo} \textbf{Entity} is intended to set up the analysis
+   As reported in Section~\ref{sub:singleRun}, the \textit{RunInfo} \textbf{Entity} is intended to set up the analysis
    that the user wants to perform. In this specific case, two steps (\xmlNode{Sequence}) are  run
-   using 12 processors (\xmlNode{batchSize}). This means that
-   12 instances of the driven code are run simultaneously.
-   Every time a simulation ends, a new one is launched.
+   using 1 processor (\xmlNode{batchSize}). This means that
+   1 instance of the driven code are run simultaneously.
```

Comment:
are -> is, but the sentence itself doesn't make sense with a single processor. Maybe "This means that a single instance of the driven code is run at a time."

Reply:
Changed